samples/midi-output/package.json

```
@@ -0,0 +1,23 @@
+{
+    "name": "midi-output",
+    "version": "0.1.0",
+    "nodecg": {
+        "compatibleRange": "^1.1.1",
+        "bundleDependencies": {
+            "nodecg-io-midi-output": "0.1.0"
+        }
+    },
+    "scripts": {
+        "build": "tsc",
+        "watch": "tsc -w",
+        "list": "node helperscripts/listDevices.js"
```

```suggestion
        "list": "node ../midi-input/helperscripts/listDevices.js"
```
It would be nice if you could start the same script in the midi-input sample here so we don't have to maintain two versions of this script.